src/Aspire.Hosting.Keycloak/KeycloakResourceBuilderExtensions.cs

```diff
@@ -59,10 +62,13 @@ public static IResourceBuilder<KeycloakResource> AddKeycloak(
             .WithImageRegistry(KeycloakContainerImageTags.Registry)
             .WithImageTag(KeycloakContainerImageTags.Tag)
             .WithHttpEndpoint(port: port, targetPort: DefaultContainerPort)
+            .WithHttpEndpoint(targetPort: HealthCheckContainerPort, name: HealthEndpointName)
```

Comment:
Consider adding the management interface port as an optional parameter too (like `port`) so it can be easily changed by the caller. If it's set then we'd need to set the `KC_HTTP_MANAGEMENT_PORT` environment variable too.

Reply:
Wouldn't that mean that we should also support `KC_HTTP_PORT`?

But maybe we could add a `managementPort` parameter with the same semantics of the `port` parameter, without messing with target ports.

Would it be feasible to determine the target ports based on the existence or not of `KC_HTTP_PORT` and `KC_HTTP_MANAGEMENT_PORT`?

Reply:
> But maybe we could add a managementPort parameter with the same semantics of the port parameter, without messing with target ports.

Yeah this is what I think I meant 😄 Just let the user specify a different management port which we'll assign to the proxy port. Target port will stay the same always (unless user manually edits the added the endpoint, which of course they can still do).

Reply:
Added with tests!